build_shared_vars.sh

```diff
@@ -21,21 +30,10 @@ Options:
 -k              keep kernel tmp after build
 -d <device>     only build the kernel for <device>
 -O <directory>  build kernel in <directory>
+-D <directory>  copy kernel/dtbo to <directory>
```

Comment:
Do not copy kernel to a custom dir (as nothing looks for it in there), only dtbo.